stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -93,19 +93,21 @@ public static void main(String[] args) {
   @Override
   public String getBenchDescription() {
     return String.join("\n", ImmutableList.of(
-        "A benchmarking tool to measure the read performance of alluxio when the client "
-        + " preforms different read operation",
-        "To run the read test, data should be first generated through operation \"write\"",
+        "A benchmarking tool to measure the end-to-end read performance of Alluxio "
+             + "when the client preforms different read operation",
+        "To run the read test, data should be first generated with the \"Write\" operation ",
         "",
         "Example:",
-        "# This test will run create a 500MB file with block size 15k on 1 worker,",
-        "# then test the ReadArray operation for 30s and record the throughput after 10s warmup.",
+        "# This test will run create a 500m file with block size 15k on 1 worker,",
```

Comment:
Here I believe `MB` and `KB` are more expressive. `m` often means `million` and `k` means `thousand`. Let's stick to `MB` and `KB` as much as possible.